langchain/src/vectorstores/base.ts

```diff
@@ -13,6 +13,8 @@ export abstract class VectorStore {
     documents: Document[]
   ): Promise<void>;
 
+  abstract addDocuments(documents: Document[]): Promise<void>;
```

Comment:
Thank you!

Reply:
why thank you? lol i had to do this

Reply:
Because you added something I had forgotten a few weeks ago :)